bionic/code_references.py

```diff
@@ -167,11 +179,7 @@ def set_tos(t):
                     Please raise a new issue at
                     https://github.com/square/bionic/issues to let us know.
                     """
-                message += """
-                You can also suppress this warning by removing the
-                `suppress_bytecode_warnings` override from the
-                `@version` decorator on the corresponding function.
-                """
+                message += make_suppression_advice(code.co_name)
```

Comment:
This will give you the function name of whatever `code` is, which can be a non-bionic entity function. The advice suggests the user to change the `@version` decorator on the function which won't work for non-bionic entity functions. If you still want to show the function name, you will have to plumb the top level function name here.

Reply:
I updated the message to make it clear that the named function is not necessarily the same as the top-level function. @namanjain WDYT?

Reply:
LGTM